module_1/first-react-app/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import SearchBox from './components/SearchBox.js';
+import MovieGenre from './components/MovieGenre.js';
+import CounterLegacy from './components/CounterLegacy.js';
+
+
+
+
+
+function App() {
+
+  useEffect(() => {
+    // Initialize Materialize collapsible when the component mounts
+    const collapsibleElems = document.querySelectorAll('.collapsible');
+    window.M.Collapsible.init(collapsibleElems, {
+      accordion: false
+    });
+
+  }, []); // Empty dependency array ensures that this effect runs once after the initial render
```

Comment:
Please remove all the comments, you can keep comments locally but in the PR's better delete them.

Reply:
Done in commit: c7a4e1e3bff35922c071c59fe3905a3da37fefef